frontend/src/assets/i18n/en.json

```diff
@@ -0,0 +1,219 @@
+{
+    "APP_NAME": "MedTagger",
+    "HOME": {
+        "DROPDOWN": {
+            "SETTINGS": "Settings",
+            "LOGOUT": "Log out"
+        },
+        "HEADER": {
+            "GREETING": "Hello",
+            "DESCRIPTION": "MedTagger Home Page"
```

Comment:
If we woul leave the APP_NAME, could we use it here? Something along the lines for
`"DESCRIPTION": "APP_NAME +  Home Page"`

Reply:
Nope, it should be concatenated in the source. As it is a description, I think its better to have it as one resource string (we can even drop MedTagger phrase here).